courses/util.py

```diff
@@ -166,6 +176,8 @@ def upsert_course_from_opendata(info, semester):
     set_meetings(section, info['meetings'])
     add_associated_sections(section, info)
     add_restrictions(section, info['requirements'])
+    relocate_reqs_from_rests(info['requirements'],
+                             info['fulfills_college_requirements'].append('Humanities & Social Science Sector'))
```

Comment:
was the `.append('Humanities & Social Science Sector')` for testing? looks like this would always add Humanities to every class.

Reply:
Wow sry abt that...